tests/contrib/mongoengine/test.py

```diff
@@ -42,8 +40,9 @@ def test_insert_update_delete_query(self):
 
         # ensure we get a drop collection span
         spans = tracer.pop()
-        assert len(spans) == 1
-        span = spans[0]
+        assert len(spans) == 2
```

Comment:
Sorry dumb question, why did this change to 2? 😅

Reply:
I made a subtle change to the pin object. I made sure the SAME pin object is applied to client, server, and socket connections. Previously the pin object used in this test had two different tracers for client and server connections. So this test was only configured to capture and assert the number of client spans. This looked wrong to me.

Now this test captures all the spans generated by the pymongo client and server objects in one tracer.